README.md

```diff
@@ -8,11 +8,12 @@
 
 ## Slides
  - [Web2day 2017](https://docs.google.com/presentation/d/1ffUXgs4dimSTOKi3RtfW_6FYeabH9mQs19S1Om2-73k/edit?usp=sharing)
+ - [BDX.io 2017](https://docs.google.com/presentation/d/1QipKxyUIcntcfenZPb6IzpMv11nKSCUyfVfB5VIgaWI/edit?usp=sharing)
```

Comment:
Don't add link like that.
This is the public version of the conference (with all history etc)
We will make a public (published) version after the conference

Reply:
:ok_hand: